Frontend/src/Components/Shared/colors.js

```diff
@@ -0,0 +1,34 @@
+const colors = {

```

Comment:
This would be an excellent place to put that function that will retrieve the value.

It would also protect against getting a light/dark mode color that doesn't exist and will pave the way for another automated check